include/tdi/common/c_frontend/tdi_table.h

```diff
@@ -741,8 +741,8 @@ tdi_status_t tdi_key_field_id_list_get(const tdi_table_hdl *table_hdl,
  * @return Status of the API call
  */
 tdi_status_t tdi_key_field_type_get(const tdi_table_hdl *table_hdl,
-                                     const tdi_id_t field_id,
-                                     tdi_key_field_type_t *field_type);
+                                    const tdi_id_t field_id,
```

Comment:
This function can probably be changed to `tdi_key_field_match_type_get` and it should return `tdi_match_type_e`